velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -1114,6 +1186,67 @@ void JoinFuzzer::addPlansWithTableScan(
         groupedProbeScanSplits,
         groupedBuildScanSplits));
   }
+
+  // Add ungrouped MergeJoin with TableScan.
+  auto planNodeIdGenerator = std::make_shared<core::PlanNodeIdGenerator>();
```

Comment:
No, grouped execution means the both input tables are already partitioned by the join key, so that one driver can join the partitions from both side on the same key value without knowing other partitions. It only supports hash join and hash aggregation according to the design doc (https://github.com/prestodb/presto/issues/12124).